packages/jokul/src/components/button/styles/button.scss

```diff
@@ -5,21 +5,21 @@
 $_flash-animation-name: jkl-tertiary-flash-#{string.unique-id()};
 
 @include jkl.comfortable-density(".jkl-button") {
-    --padding-block: #{jkl.$spacing-8};
-    --padding-text: #{jkl.$spacing-24};
-    --padding-icon: #{jkl.$spacing-16};
-    --padding-icon-button: #{jkl.$spacing-8};
-    --padding-tertiary-inline: #{jkl.$spacing-4};
-    --padding-ghost-inline: #{jkl.$spacing-8};
+    --padding-block: #{jkl.$unit-10};
+    --padding-text: #{jkl.$unit-30};
+    --padding-icon: #{jkl.$unit-20};
+    --padding-icon-button: #{jkl.$unit-10};
+    --padding-tertiary-inline: #{jkl.$unit-02};
+    --padding-ghost-inline: #{jkl.$unit-10};
 }
 
 @include jkl.compact-density(".jkl-button") {
-    --padding-block: #{jkl.$spacing-4};
-    --padding-text: #{jkl.$spacing-12};
-    --padding-icon: #{jkl.$spacing-8};
-    --padding-icon-button: #{jkl.$spacing-4};
-    --padding-tertiary-inline: #{jkl.$spacing-2};
-    --padding-ghost-inline: #{jkl.$spacing-4};
+    --padding-block: #{jkl.$unit-02};
+    --padding-text: #{jkl.$unit-15};
+    --padding-icon: #{jkl.$unit-10};
+    --padding-icon-button: #{jkl.$unit-02};
+    --padding-tertiary-inline: #{jkl.$spacing-2}; // hva tenker vi her? 2px spacing?
```

Comment:
Jeg stemmer for `unit-04` da, siden `unit-02` er 4

Reply:
vi hadde visst en unit-02 som er 2 :) 
unit-10 = 8px
unit-05 = 4px
unit-02 = 2px